.gitignore

```diff
@@ -1,6 +1,7 @@
 coverage/
 node_modules/
 .idea
+.vscode
```

Comment:
This is fine, but FYI you can add `.vscode` to your `.gitignore` file in your home directory (assuming you develop projects in subdirectories of your home directory) and it will apply to everything everywhere and you'll never need to add this to a project `.gitignore` file again.

Reply:
Good point, I actually didn't knew that!